### ISSUE TITLE
Zooming onto a location with CTRL + Mouse Wheel is broken

### ISSUE THREAD

**Issue by Sethur** [imagej/imagej#139]
Hi,

in the current development release of ImageJ2, zooming with holding down CTRL and using the mousewheel does not seem to aim on the targeted pixel any longer (as it does in the current version of ImageJ1). When using "+" and "-", the aim is correct.

**Comment by ctrueden** [comment]
Thanks for the report. Yes, this is a known issue with the ImageJ2 Swing user interface right now. It used to work, but sadly, got broken at some point. Note that the ImageJ2 Swing UI is currently dormant, as development efforts are focused on ImageJ Ops and the ImageJ2 data model. The ImageJ 1.x legacy UI is currently the default UI for ImageJ2 and Fiji, and will likely remain so for quite some time. It is not clear whether we will resurrect the Swing UI, or pursue a new UI which deviates more from the original ImageJ UI in favor of more recent UI sensibilities such as JavaFX.

**Comment by Sethur** [comment]
I have created a pull request to fix this issue on ImageJA

**Comment by ctrueden** [comment]
Thanks @Sethur! To clarify, my comments above are regarding the ImageJ2 Swing UI, not the ImageJ 1.x AWT UI, which is what you fixed with imagej/ImageJA#12. FYI, I believe @rasband is traveling right now, so there might be some delay before he responds.

**Comment by Sethur** [comment]
@ctrueden No big deal. Is the IJ2 Swing UI still dormant? I think Fiji currently also uses the AWT one, but I might be wrong there.

**Comment by ctrueden** [comment]
> Is the IJ2 Swing UI still dormant?

Yes, indefinitely. It is possible we will develop it further, but it seems many users would prefer something different and more "modern" such as [ImageJFX](https://imagej.net/ImageJFX).

> I think Fiji currently also uses the AWT one

Yes, Fiji uses the "ImageJ Legacy" UI—i.e., the ImageJ 1.x AWT UI—by default. However, it does also ship with the IJ2 Swing UI, and can be launched in that mode, via the _Help &#9654; Switch to Modern Mode_ command. Note that we are [considering removing this command](https://github.com/imagej/imagej-legacy/issues/115), though, in favor of a setting in the ImageJ2 options dialog or similar. Note also that there are currently major bugs surrounding this functionality (https://github.com/fiji/fiji/issues/71, https://github.com/imagej/imagej-legacy/issues/25).
